src/client/modules/my/fivePlayerSummaryGraphs/fivePlayerSummaryGraphs.ts

```diff
@@ -40,6 +46,28 @@ export default class SummaryGraphs extends LightningElement {
                     }
                     return 0;
                 });
+            // Games Played Most Frequent
+            this.gamesPlayedDonuntData = playerOverviewStats
+                .filter(ps => ps.num_games > 1)
+                .map((ps) => { return {name: ps.player_name, value: ps.num_games}; })
+                .sort((a, b) => {
+                    if (a.value < b.value) {
+                        return -1;
+                    } else if (a.value > b.value) {
+                        return 1;
+                    }
+                    return 0;
+                });
+            // Average Points
+            this.avgPointsWonBarData = playerOverviewStats
+                .sort((a,b) => {
```

Comment:
Does this sort the data such that highest avg is on the left, descending to the right?

Reply:
Yes